Instance/config.py

```diff
@@ -1,18 +1,21 @@
 import os
 '''configuration options'''
 
+
 class Config(object):
     '''Parent configuration class'''
     DEBUG = False
     SECRET_KEY = 'tryandguess'
     ENV = 'development'
     SECRET = os.getenv('SECRET')
```

Comment:
You have a `SECRET_KEY` and a `SECRET` from the environment. You should pick one, preferably one that is hidden in the env.